src/Data/Lens/At.purs

```diff
@@ -60,3 +61,6 @@ instance atForeignObject :: At (FO.Object v) String v where
   at k =
     lens (FO.lookup k) \m ->
       maybe' (\_ -> FO.delete k m) \v -> FO.insert k v m
+
+sans :: forall m a b. At m a b => a -> m -> m
+sans k = at k .~ Nothing
```

Comment:
Would you mind using `set` instead of the `.~` operator here? I'd like to try and use the named functions in implementations where possible, as the operators aren't used widely the way they are in Haskell.